### DISCUSSION TITLE
如何通过最小化改动，实现layui通过nginx部署到二级目录下？

### DISCUSSION THREAD

**Discussion by szxypi** [layui/layui#2093] [问答]
例如：主域名是 https://xxx.com。想实现layui前端部署后访问路径为 https://xxx.com/abc
页面引用js css image 可以改成相对路径来实现，但是左侧菜单和页面之间的跳转需要怎么调整？恳请大佬提供解决方案。

**Comment by sentsim** [comment]
这个跟 Layui 没有太大关系。
直接在 Nginx 配置好 location，前端页面只是做对应的路径引入。
